syntax = "proto3";

package tone;

service ToneService {
    rpc GetTone (GetToneRequest) returns (GetToneResponse);
}

message GetToneRequest {
    string text = 1;
}

message GetToneResponse {
    Tone tone = 1;
}

enum Tone {
    TONE_UNSPECIFIED = 0;
    TONE_POSITIVE = 1;
    TONE_NEGATIVE = 2;
}